### PR TITLE
25 make a new step by step guide for new users of the template

### PR DESCRIPTION
**Pull request by janhalen**
Complete rewrite to make it easier to get started with the template.

The guide is now divided into a Maintainer and a Contributor part.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,45 @@
...
 
-- uses the [Just the Docs][Just the Docs] theme;
-- can be built and published on [GitHub Pages][GitHub Pages];
-- can be built and previewed locally, and published on other platforms.
+*In OS2 projects the maintainer team is the group of individuals assigned to oversee the development, quality, and community engagement of the documentation project.*
 
-[Browse documentation][Just the Docs] to learn more about how to use this theme.
+# Setup
+
+1. Start by [creating a documentation repository in your project, from this template](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template), including only the main branch.
```
Comment (zorp) [inline README.md:11]: The link should point to the actual template repro. Or am I misreading the sentence "from this template".

File: README.md [hunk 2]
```diff
@@ -1,14 +1,45 @@
...
+3. Configure a [publishing source for GitHub Pages](https://help.github.com/en/articles/configuring-a-publishing-source-for-github-pages). The documentation site is now live!
```
Comment (zorp) [inline README.md:12]: Is it clear for the noob what to do in the actual repro. It's seems simple from reading the guide but have the steps been tested and verified when performing on a new repro created from the OS2 documentation template?

File: README.md (hunk 1 above)
Comment (zorp) [reply README.md:11]: Sorry I forgot that you the person reading this is actually inside the template and will have the button "use this template" available.

File: README.md [hunk 3]
```diff
@@ -1,14 +1,45 @@
...
+4. Edit and create `.md` [Markdown files](https://guides.github.com/features/mastering-markdown/) to add more content pages to your docs.
```
Comment (zorp) [inline README.md:13]: For this to be clear the template must contain the bare minimum of .md files to be filled in. Else my fear is that no additional files are created.

**Review by zorp — COMMENTED** [review]
I think it's a simple ad easy to access guide. I have added a few comments, they are not critical - just discussion point.

File: README.md (hunk 3 above)
Comment (janhalen) [reply README.md:13]: I suggest we define a minimum set of files (e.g. "how_to_deploy") and reference these files to Foundation for Public code governance.

**Comment by janhalen** [comment]
@zorp the README template has been restructured to include a step by step guide in the top. The idea is to offer the standard setup instructions inside the README template to make it fast and simple to get started:

https://github.com/OS2offdig/os2-docs-template/blob/25-make-a-new-step-by-step-guide-for-new-users-of-the-template/README.md

Any comments before i merge?

**Review by os2git — DISMISSED** [review]
Great work. Time to merge

**Review by zorp — APPROVED** [review]
Great work. Time to merge.

**Pull request merged**